deploy-sg-lambda.yml

```diff
@@ -13,90 +13,18 @@
 
 
 - hosts: localhost
-  vars_files:
-    - aws_credentials.yml
   vars:
+    aws_account_name: build
     ansible_connection: local
     lambda_name: cf-security-group-update
-    lambda_function_name: cf-security-group-update-test
+    lambda_function_name: cf-security-group-update
     lambda_handler: "{{lambda_name}}.lambda_handler"
-    security_group_name: cloudflare_lambda_demo_sg
+    security_group_name: cloudflare_lambda_sg
     output_dir: ~/ansible_testing
+    lambda_role: "arn:aws:iam::{{aws_account}}:role/cloudflare_sg_lambda"
```

Comment:
good improvement on having the aws account as a env. var than an hardcoded value.